ckanext/dcatapit/i18n/de/LC_MESSAGES/ckanext-dcatapit.po

```diff
@@ -693,56 +685,46 @@ msgid "Add new Creator"
 msgstr "Schöpfer hinzufügen"
 
 #: ckanext/dcatapit/templates/macros/dcatapit_form_macros.html:233
-#, fuzzy
 msgid "Add new Temporal Coverage"
 msgstr "zeitliche Abdeckung"
 
 #: ckanext/dcatapit/templates/macros/dcatapit_form_macros.html:255
 #: ckanext/dcatapit/templates/package/snippets/dcatapit_additional_info.html:87
-#, fuzzy
 msgid "OthId Identifier"
 msgstr "anderen Bezeichner"
 
 #: ckanext/dcatapit/templates/macros/dcatapit_form_macros.html:256
-#, fuzzy
 msgid "OthId dataset identifier"
 msgstr "dataset identifier"
 
 #: ckanext/dcatapit/templates/macros/dcatapit_form_macros.html:260
 #: ckanext/dcatapit/templates/package/snippets/dcatapit_additional_info.html:90
-#, fuzzy
 msgid "OthId Organization"
 msgstr "Organisation E-Mail"
 
 #: ckanext/dcatapit/templates/macros/dcatapit_form_macros.html:261
-#, fuzzy
 msgid "OthId organization name"
 msgstr "Organisation E-Mail"
 
 #: ckanext/dcatapit/templates/macros/dcatapit_form_macros.html:265
 #: ckanext/dcatapit/templates/package/snippets/dcatapit_additional_info.html:93
-#, fuzzy
 msgid "OthId Organization ID"
 msgstr "Organisation E-Mail"
 
 #: ckanext/dcatapit/templates/macros/dcatapit_form_macros.html:266
-#, fuzzy
 msgid "OthId organization identifier"
 msgstr "Organisation Website-URL"
```

Comment:
the same as reported above 

Are these translations used? If not please remove them.

Reply:
They are in use. Lineid shows where:

https://github.com/geosolutions-it/ckanext-dcatapit/blob/master/ckanext/dcatapit/templates/macros/dcatapit_form_macros.html#L251-L272

which is displayed in alternate identifier fields:

![image](https://user-images.githubusercontent.com/116021/41713331-198d0ff2-754d-11e8-8ce3-ed34e849a5d9.png)


Also, some of them are in use in displaying those values

https://github.com/geosolutions-it/ckanext-dcatapit/blob/master/ckanext/dcatapit/templates/package/snippets/dcatapit_additional_info.html#L84-L97